dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpChannelConfig.java

```diff
@@ -2208,15 +2191,12 @@ private void parseAutoDecompression(Map<Object, Object> props) {
      *
      * @param props
      */
-    private void parsev0CookieDateRFC1123compat(Map<?, ?> props) {
-
-        Object value = props.get(HttpConfigConstants.PROPNAME_V0_COOKIE_RFC1123_COMPAT);
-        if (null != value) {
-            this.v0CookieDateRFC1123compat = convertBoolean(value);
-        }
-
-        if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
-            Tr.event(tc, "Config: v0CookieDateRFC1123compat is " + isv0CookieDateRFC1123compat() + " this = " + this);
+    protected void parsev0CookieDateRFC1123compat(Object option) {
+        if (Objects.nonNull(option)) {
+            this.v0CookieDateRFC1123compat = convertBoolean(option);
+            if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
```

Comment:
Trace message is now in the null check where before it wasn't. No need to fix right now but it is a difference on how things are logged

Reply:
This applies to numerous parse methods.